From a61dc72035c7ae0b06f6d7dc8b2a848ffc7db277 Mon Sep 17 00:00:00 2001
From: MORITA Shintaro <shin@sysphonic.com>
Date: Sat, 4 Jul 2015 09:40:26 +0900
Subject: [PATCH] x ApplicationHelper.get_sql_like() -> o
 SqlHelper.get_sql_like() Signed-off-by: MORITA Shintaro <shin@sysphonic.com>

---
 app/controllers/equipment_controller.rb | 6 +++---
 app/controllers/groups_controller.rb    | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/app/controllers/equipment_controller.rb b/app/controllers/equipment_controller.rb
index dc9e438..dc21966 100644
--- a/app/controllers/equipment_controller.rb
+++ b/app/controllers/equipment_controller.rb
@@ -143,7 +143,7 @@ def list
       if @group_id == '0'
         con << "((groups like '%|0|%') or (groups is null))"
       else
-        con << ApplicationHelper.get_sql_like([:groups], "|#{@group_id}|")
+        con << SqlHelper.get_sql_like([:groups], "|#{@group_id}|")
       end
     end
 
@@ -219,11 +219,11 @@ def schedule_all
       case display_type
        when 'group'
         if @login_user.get_groups_a(true).include?(display_id)
-          con = ApplicationHelper.get_sql_like([:groups], "|#{display_id}|")
+          con = SqlHelper.get_sql_like([:groups], "|#{display_id}|")
         end
        when 'team'
         if @login_user.get_teams_a.include?(display_id)
-          con = ApplicationHelper.get_sql_like([:teams], "|#{display_id}|")
+          con = SqlHelper.get_sql_like([:teams], "|#{display_id}|")
         end
       end
     end
diff --git a/app/controllers/groups_controller.rb b/app/controllers/groups_controller.rb
index 3fd81e5..7dc095f 100644
--- a/app/controllers/groups_controller.rb
+++ b/app/controllers/groups_controller.rb
@@ -263,7 +263,7 @@ def get_users
       if @group_id == '0'
         con << "((groups like '%|0|%') or (groups is null))"
       else
-        con << ApplicationHelper.get_sql_like([:groups], "|#{@group_id}|")
+        con << SqlHelper.get_sql_like([:groups], "|#{@group_id}|")
       end
     end
 
